public/js/common/react_modal.js

```diff
@@ -13,93 +15,80 @@ export var Modal = React.createClass({
         return {
             courseId: '',
             course: [],
-            sessions: []
+            sessions: [],
+            modalIsOpen: false,
+            courseTitle : '',
+            formatCourse : '',
```

Comment:
While you are doing *something* like that, notice that you never actually access `this.state.formatCourse`.